impeller/entity/contents/vertices_contents_unittests.cc

```diff
@@ -64,12 +65,14 @@ TEST_P(EntityTest, RendersDstPerColorWithAlpha) {
       *GetContentContext()->GetRenderTargetCache(), {100, 100},
       /*mip_count=*/1);
   auto render_pass = buffer->CreateRenderPass(render_target);
+  auto recording_pass = std::make_shared<RecordingRenderPass>(
+      render_pass, GetContext(), render_target);
   Entity entity;
 
-  ASSERT_TRUE(render_pass->GetCommands().empty());
-  ASSERT_TRUE(contents->Render(*content_context, entity, *render_pass));
+  ASSERT_TRUE(recording_pass->GetCommands().empty());
+  ASSERT_TRUE(contents->Render(*content_context, entity, *recording_pass));
 
-  const auto& cmd = render_pass->GetCommands()[0];
+  const auto& cmd = recording_pass->GetCommands()[0];
```

Comment:
nit: assert command list is not empty before indexing

Reply:
Done